index.html

```diff
@@ -15,13 +15,104 @@
 </head>
 
 <body>
+    
 
     <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+      <div class="row nav-bar">
+          <ul class="nav nav-pills nav-fill col-12">
+              <li class="nav-item">
+                <a class="nav-link active" href="#">My Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">All About Me</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Portfolio</a>
+              </li>
+            </ul>
           </div>
-      </div>
+      
+
+      <div class="row main-pic">
+          <div class="jumbotron 12">
+              <img src="img/joe.jpg" class="img-fluid" alt="Pic of Joe">
```

Comment:
Great choice here with the black and white photo.  Looks great with the styling you did.